frontend/packages/data-portal/app/components/Filters/RegexFilter.tsx

```diff
@@ -0,0 +1,96 @@
+import { useSearchParams } from '@remix-run/react'
+import { useMemo, useState } from 'react'
+
+import { QueryParams } from 'app/constants/query'
+import { useI18n } from 'app/hooks/useI18n'
+
+import { DropdownFilterButton } from './DropdownFilterButton'
+import { InputFilter } from './InputFilter'
+
+export function RegexFilter({
```

Comment:
Wonder if we should make this component more generic, cuz now we don't have a general component that's just an input field with an "Apply" right? Maybe `regex` should be optional? (could just be a TODO too)